samples/kubeflow/README.md

```diff
@@ -1,62 +1,110 @@
 
 ## Installation
 
-Run the following script to install The Mesh for Data and all dependencies required for this demo. It will take a while for all containers to be ready due to image downloads.
+Run the following script to install The Mesh for Data core components and third party dependencies required for this sample.
+Please note that it may take a while for the M4D containers to be ready as the container images may need to be downloaded.
 
+```bash
+kubectl config set-context --current --namespace=m4d-system
+```
+
+Install on OpenShift:
+```bash
+WITHOUT_OPENSHIFT=false ./hack/install.sh
+```
+
+**Or**
 
+Install on Kind:
 ```bash
-kind create cluster
-./install/install.sh
+./hack/install.sh
+```
+
+Install kubeflow on your cluster.
+On a Kind cluster for example you can use the following script to install it:
+```bash
+cd samples/kubeflow/install/kubeflow
+./install_kubeflow.sh
+
+cd -
+```
+
+Install the Arrow-Flight module
+```
+kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data-flight-module/master/module.yaml
 ```
 
 ## Data owner instructions
 
-We already have transactions.csv stored in COS (TODO: don't assume that).
-Register the asset using the following command:
+1. Upload the file `data.csv` to an object-storage of your choice. For example, IBM's COS.
+`data.csv` contains the first 100 rows from the following [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data) created by NTNU, and it is shared under the ***CC BY-SA 4.0*** license.
+
+2. Alter the `fullPath` field in the json `example_transactions.csv.json` to contain the details of your object storage location, etc.
+See the comments in [third_pary/egeria/usage/create_new_asset.sh](../../third_party/egeria/usage/create_new_asset.sh) for more details.
+
+3. Register the asset using the following command:
 
 ```bash
+
+cd third_party/egeria/usage
+
+# Wait for the port-forward to take effect before proceeding to the next command
 kubectl port-forward -n egeria-catalog svc/lab-core 9443:9443 &
-sleep 5
-cd dataowner
-./create_new_asset.sh transactions.csv.json 'finance'
+
+
+# path-to-transactions.csv.json can be for example ../../../samples/kubeflow/example_transactions.csv.json for using the example file
+./create_new_asset.sh <path-to-transactions.csv.json> 'finance'
+
 cd -
 kill $!
 ```
 
-Note the asset ID that is listed next to "Read asset info: ". You will need it later:
+4. Save the asset ID (example for asset-id is 5de27155-48d3-4d78-8767-73e7b264e394).
+Export it to environment variable:
 ```
 export ASSET_ID=<asset-id>
 ```
 
-The credentials for the asset need to be registered in Vault. 
-You can do that using the Vault UI. First use this command to be able to open the UI from the browser:
+5. Register the credentials for the dataset in Vault. Currently, only hmac credentials are supported.
+Due to consistency with internal systems, currently the terms `access_key` and `secret_key` are used.
```

Comment:
```suggestion
5. Register in Vault the S3 credentials `access_key` and `secret_key` required to access the dataset . 
```

Reply:
I pushed a newer version with (just) saying aka `access_key_id` and `secret_access_key`